velox/docs/configs.rst

```diff
@@ -164,4 +164,29 @@ data size exceeds this limit, otherwise it spills the partition with most data.
 If the limit is zero, then the spiller always spills a previously spilled
 partition if it has any data. This is to avoid spill from a partition with a
 small amount of data which might result in generating too many small spilled
-files.
\ No newline at end of file
+files.
+
+
+Hive Connector
+-----------------------------
+
+``max_partitions_per_writers``
```

Comment:
As discussed, let's add this in the same PR that starts using it.

Reply:
Forgot this place